azurerm/internal/services/storage/storage_account_resource.go

```diff
@@ -1243,7 +1328,12 @@ func resourceStorageAccountRead(d *schema.ResourceData, meta interface{}) error
 			}
 		}
 
-		if err := d.Set("blob_properties", flattenBlobProperties(blobProps)); err != nil {
+		blobServiceProps, err := accountsClient.GetServiceProperties(ctx, name)
```

Comment:
and why are we doing a get here?

Reply:
So that we can flatten and set the `"blob_properties"` attribute. I should probably remove `blobProps` above and the error check.